Event/PasswordResetTokenCreated.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Event;
+
+use SumoCoders\FrameworkMultiUserBundle\User\PasswordReset;
+use Symfony\Component\EventDispatcher\Event;
+
+class PasswordResetTokenCreated extends Event
+{
+    const NAME = 'multi_user.event.password_reset_token_created';
+
+    /**
+     * @var PasswordReset
+     */
+    private $user;
+
+    /**
+     * PasswordResetTokenCreated constructor.
+     *
+     * @param PasswordReset $user
+     */
+    public function __construct(PasswordReset $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * @return PasswordReset
```

Comment:
ik zie net dat die user van het type passwordreset is volgens je phpdoc, maar dan zou ik die variabele anders noemen, da ziet er zo maar raar uit